### DISCUSSION TITLE
Telling a creator that their files aren't working

### DISCUSSION THREAD

**Discussion by Azeyyy** [community/community#22570] [New to GitHub]
<p>I don’t have much knowledge on coding or much to do with anything around that are, but files i downloaded didn’t work for me, or my friend, and they did before the update, I tried messaging the creator and or making a pull but i didnt find any way to message him or a way i could tell him</p>

**Comment by Wabri** [comment, marked as answer]
<p>Is the project on github? If the answer is yes you can create a new issue where you explain your problem. To do that, you have to click in the “issues” tab:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6ade8cd5036c17657f1fb9e3973e9805c8c226fc.png" data-download-href="/uploads/short-url/ffpqfd7wz33PTgZqRvFsZf6QpHe.png?dl=1" title="issues.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181072021-82710f3e-f275-422b-9e28-356769c9579e.png" alt="issues.png" data-base62-sha1="ffpqfd7wz33PTgZqRvFsZf6QpHe" width="690" height="396" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/6ade8cd5036c17657f1fb9e3973e9805c8c226fc_2_690x396.png, https://aws1.discourse-cdn.com/github/original/2X/6/6ade8cd5036c17657f1fb9e3973e9805c8c226fc.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/6/6ade8cd5036c17657f1fb9e3973e9805c8c226fc.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6ade8cd5036c17657f1fb9e3973e9805c8c226fc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">issues.png</span><span class="informations">702×403 40.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Than click on “new issue”:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/1357910e03ee86c2c6e1db82686aca64676e2ea6.png" data-download-href="/uploads/short-url/2L6GjKpwgzvSm494aUcvql29yAK.png?dl=1" title="than.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181072022-bc595fe7-fd6f-4c4c-8a14-e1f4b482df8f.png" alt="than.png" data-base62-sha1="2L6GjKpwgzvSm494aUcvql29yAK" width="690" height="173" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/1357910e03ee86c2c6e1db82686aca64676e2ea6_2_690x173.png, https://aws1.discourse-cdn.com/github/optimized/2X/1/1357910e03ee86c2c6e1db82686aca64676e2ea6_2_1035x259.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/1/1357910e03ee86c2c6e1db82686aca64676e2ea6_2_1380x346.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/1357910e03ee86c2c6e1db82686aca64676e2ea6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">than.png</span><span class="informations">1774×446 68.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Try to summarize the problem in the title and in the “write” section explain all the problems you had.</p>
<p>Hope to help you.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/azeyyy">@azeyyy</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our community organized so users can more easily find information.</p>
<p>As you’ll notice, your topic is now here in the Project Development Help and Advice board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>
<p>Andrea</p>
